app/controllers/concerns/course/lesson_plan/strategies/fomo_personalization_strategy.rb

```diff
@@ -80,15 +80,27 @@ def cannot_shift_item(course_user, item, submitted_items, items_to_shift)
       (!items_to_shift.nil? && !items_to_shift.include?(item.id))
   end
 
+  def item_is_straggling(personal_time, reference_time)
+    if reference_time.end_at.present? && personal_time.end_at.present?
+      reference_time.end_at < personal_time.end_at
+    elsif reference_time.end_at.present? && personal_time.end_at.nil?
```

Comment:
We may need to revisit this.. for unsubmitted items, personal_time might be recomputed (and removed) if the new reference time of a new reference timeline is created with no `end_at`